data/styles/adwaita.mplstyle

```diff
@@ -0,0 +1,61 @@
+# adwaita
+patch.facecolor: FAFAFA
+patch.linewidth: .3
+
+axes.axisbelow: True
+axes.edgecolor: black
+axes.facecolor: white
+axes.grid: True
+axes.labelcolor: black
+axes.labelsize: 11
```

Comment:
I think this maybe the line that overrides the chosen font size. If we want to use 11 as default font size, we can set that in the config file.